### DISCUSSION TITLE
API for Openwebui

### DISCUSSION THREAD

**Discussion by alesha-pro** [open-webui/open-webui#5960] [Ideas]
Do the developers have any plans to implement their own API? What do I mean by that?

I would like to see Openwebui also as a powerful backend for api implementation. I can aggregrate all my providers, my pipelines, documents and RAG search inside Openwebui and would like to have a single endpoint for my other applications. I could customize this all internally and use it as a backend. 

Have there been any ideas like this? This functionality would be really nice.

**Comment by tjbck** [comment]
https://docs.openwebui.com/getting-started/api
